src/main/java/com/limechain/runtime/hostapi/functions/HostFunctions.java

```diff
@@ -0,0 +1,37 @@
+package com.limechain.runtime.hostapi.functions;
+
+import lombok.experimental.UtilityClass;
+import org.wasmer.ImportObject;
+import org.wasmer.Type;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.function.UnaryOperator;
+
+@UtilityClass
+public class HostFunctions {
+
+    static final List<Number> EMPTY_LIST_OF_NUMBER = List.of();
+    static final List<Type> EMPTY_LIST_OF_TYPES = List.of();
```

Comment:
Be specific about the access modifier to be consistent with other parts of the code.